### PR TITLE
Don't use cd in setup/path.sh

### PR DESCRIPTION
**Pull request by lmoureaux**
Addresses bug #109 assuming that $BUILD_HOME is an absolute path.

### CODE REVIEW

File: setup/paths.sh [hunk 1]
```diff
@@ -38,18 +38,14 @@ else
...
+export "PYTHONPATH=$PYTHONPATH:$BUILD_HOME/vfatqc-python-scripts"
 
-cd $BUILD_HOME/vfatqc-python-scripts/setup
-export PATH=$PATH:$PWD
+export "PATH=$PATH:$BUILD_HOME/vfatqc-python-scripts/setup"
 
 # Done
-cd $BUILD_HOME/vfatqc-python-scripts
-export GEM_VFATQC_PROJECT=$PWD
+export "GEM_VFATQC_PROJECT=$BUILD_HOME/vfatqc-python-scripts"
```
Comment (cbravo135) [inline setup/paths.sh]: Isn't it cleaner to do GEM_VFATQC_PROJECT first and then use it to make the others. This all could be done in 3 simpler lines.

Comment (jsturdy) [reply setup/paths.sh]: Yeah, probably.
Also what is `GEM_VFATQC_PROJECT` actually used for?

**Review by jsturdy — CHANGES_REQUESTED** [review]

File: setup/paths.sh [hunk 2]
```diff
@@ -38,18 +38,14 @@ else
 fi
 
 # Adding Scan Applications to Path
-#cd $BUILD_HOME/cmsgemos/gempython/tools
-#export PYTHONPATH=$PYTHONPATH:$PWD
+#export "PYTHONPATH=$PYTHONPATH:$BUILD_HOME/cmsgemos/gempython/tools"
```
Comment (bdorney) [inline setup/paths.sh]: shouldn't this be uncommented?

**Review by bdorney — CHANGES_REQUESTED** [review]
Agree w/@cbravo135 regarding cleaning up setup scripts.  Probably the same could be done in gem-plotting-tools

File: setup/paths.sh (hunk 1 above)
Comment (lmoureaux) [reply setup/paths.sh]: I didn't find any use of `GEM_VFATQC_PROJECT` in `vfatqc_python_tools`, `cmsgemos` and `gem-plotting-tools`

File: setup/paths.sh (hunk 2 above)
Comment (lmoureaux) [reply setup/paths.sh]: I didn't want to introduce any breaking change.

Comment (jsturdy) [reply setup/paths.sh]: * it was commented in the previous version (according to the diff in this PR)
* also no, the setup script in `cmsgemos` should appropriately set the `PYTHONPATH` for use with the `cmsgemos` code, if it doesn't (it [does)](https://github.com/cms-gem-daq-project/cmsgemos/blob/generic-amc-release-v3/setup/paths.sh#L67) then open an issue there

File: setup/paths.sh (hunk 1 above)
Comment (jsturdy) [reply setup/paths.sh]: In that case, then you can make it a script local variable (if even at all), i.e., don't `export` it

**Review by bdorney — APPROVED** [review]

File: setup/paths.sh [hunk 3]
```diff
@@ -38,18 +38,12 @@ else
...
 
-cd $BUILD_HOME/vfatqc-python-scripts
-export PATH=$PATH:$PWD
-export PYTHONPATH=$PYTHONPATH:$PWD
-
-cd $BUILD_HOME/vfatqc-python-scripts/setup
-export PATH=$PATH:$PWD
+export "PATH=$PATH:$BUILD_HOME/vfatqc-python-scripts"
+export "PATH=$PATH:$BUILD_HOME/vfatqc-python-scripts/setup"
```
Comment (jsturdy) [inline setup/paths.sh]: Line 44 should be removed unless there is a good reason to keep it

File: setup/paths.sh [hunk 4]
```diff
@@ -38,18 +38,12 @@ else
...
+export "PYTHONPATH=$PYTHONPATH:$BUILD_HOME/vfatqc-python-scripts"
 
 # Done
-cd $BUILD_HOME/vfatqc-python-scripts
-export GEM_VFATQC_PROJECT=$PWD
 echo GEM_VFATQC_PROJECT $GEM_VFATQC_PROJECT
```
Comment (jsturdy) [inline setup/paths.sh]: As per comments, if `GEM_VFATQC_PROJECT` isn't used outside this script, don't `export` it and remove line 48

File: setup/paths.sh (hunk 2 above)
Comment (jsturdy) [reply setup/paths.sh]: As is, just remove line 41

**Review by jsturdy — CHANGES_REQUESTED** [review]
* Prefer sandboxing of all setup scripts
 * Change  https://github.com/lmoureaux/vfatqc-python-scripts/blob/d97e2a0ae35ad541e2d48730271bcd24a2a90fe5/setup/paths.sh#L22-L29 to instruct the user to ensure that `
$BUILD_HOME/cmsgemos/setup/paths.sh` gets sourced
 * Same for https://github.com/lmoureaux/vfatqc-python-scripts/blob/d97e2a0ae35ad541e2d48730271bcd24a2a90fe5/setup/paths.sh#L31-L38
* Don't arbitrarily add locations to the `$PATH`, this will have negative consequences

**Review by jsturdy — APPROVED** [review]

**Review by jsturdy — APPROVED** [review]
Closes #109

**Pull request merged**